### PR TITLE
configure spring-boot actuator endpoint.

### PR DESCRIPTION
**Pull request by bbortt**
I think this should do it. please review the named ports.
@mirioeggmann can you try applying the configuration like this?

### CODE REVIEW

**Comment by mirioeggmann** [comment]
@bbortt generally works like a charm :) thank you!
there is just one little thing that you have to add, I will point that out in a sec

**Comment by mirioeggmann** [comment]
I had to change the actuator port from 8090 to 8080 (and change some settings in the readinessProbe etc.) for GKE to be able to perform the health check on /actuator/health. Otherwise the ingress would not start because it thinks that the deployment is not healthy. I granted you access on the cluster. Maybe try it on your own if you can add some configuration to the ingress so you can let it check on 8090/actuator/health.

File: backend/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ FROM openjdk:11-jre-slim
 RUN useradd -ms /bin/bash haubzit
 
 EXPOSE 8080
-EXPOSE 8081
+EXPOSE 8090
```
Comment (bbortt) [inline backend/Dockerfile]: this is no longer necessary, then.

File: backend/src/main/resources/config/application.yml [hunk 2]
```diff
@@ -13,3 +13,6 @@ spring:
     url: ${APPL_DATASOURCE_JDBC_URL}
     user: ${APPL_DATASOURCE_USERNAME}
     password: ${APPL_DATASOURCE_PASSWORD}
+#management:
+#  server:
+#    port: 8090
```
Comment (bbortt) [inline backend/src/main/resources/config/application.yml]: remove.

File: k8s/deployment-backend.yaml [hunk 3]
```diff
@@ -36,13 +36,12 @@ spec:
             - name: rest
               containerPort: 8080
             - name: actuator
-              containerPort: 8081
+              containerPort: 8090
```
Comment (bbortt) [inline k8s/deployment-backend.yaml]: same as rest.

**Review by bbortt — COMMENTED** [review]
cleanup actuator port.

**Comment by bbortt** [comment]
@mirioeggmann aight, I think I understand. GKE does not actually use an ingress, but a firewall to expose our service. therefore this special `readinessProbe` thingy. what about [this](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress?hl=de#direct_hc)?
```
apiVersion: cloud.google.com/v1
kind: BackendConfig
metadata:
  name: http-hc-config
spec:
  healthCheck:
    checkIntervalSec: 15
    port: 15020
    type: HTTPS
    requestPath: /healthz
```

**Review by mirioeggmann — APPROVED** [review]
@bbortt cleaned the configs and added a issue for the suggested improvement https://github.com/yb-hackathon-code-butcher/codebase/issues/78

**Pull request merged**